File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/main/Identifier.java
package pize.tests.voxelgame.main;

public class Identifier{

    public static final String DEFAULT_NAMESPACE = "minecraft";


    public static String namespaceID(String namespace, String id){
        return namespace + ":" + id;
    }

    public static String namespaceID(String id){
        return namespaceID(DEFAULT_NAMESPACE, id);
    }

}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/main/Identifier.kt
package pize.tests.voxelgame.main

object Identifier {
    const val DEFAULT_NAMESPACE = "minecraft"
    fun namespaceID(namespace: String, id: String): String {
        return "$namespace:$id"
    }

    fun namespaceID(id: String): String {
        return namespaceID(DEFAULT_NAMESPACE, id)
    }
}
